packages/arepa-wallet/src/lib/arepa-wallet.ts

```diff
@@ -0,0 +1,271 @@
+import * as nearAPI from "near-api-js";
+import type {
+  WalletModuleFactory,
+  WalletBehaviourFactory,
+  BrowserWallet,
+  Transaction,
+  Optional,
+  Network,
+  Account,
+} from "@near-wallet-selector/core";
+import { createAction } from "@near-wallet-selector/wallet-utils";
+import icon from "./icon";
+
+export interface ArepaWalletParams {
+  walletUrl?: string;
+  iconUrl?: string;
+  deprecated?: boolean;
+  successUrl?: string;
+  failureUrl?: string;
+}
+
+interface ArepaWalletState {
+  wallet: nearAPI.WalletConnection;
+  keyStore: nearAPI.keyStores.BrowserLocalStorageKeyStore;
+}
+
+interface ArepaWalletExtraOptions {
+  walletUrl: string;
+}
+
+const resolveWalletUrl = (network: Network, walletUrl?: string) => {
+  if (walletUrl) {
+    return walletUrl;
+  }
+
+  switch (network.networkId) {
+    case "mainnet":
+      return "https://mi.arepa.digital";
+    case "testnet":
+      return "https://testnet.arepa.digital";
```

Comment:
I tried this URL and it set up a wallet on mainnet, not testnet!

Reply:
Sorry the testnet is already up, thanks

Reply:
Signing up on that URL gave me 0.2 NEAR on mainnet (https://nearblocks.io/txns/EQn8NVKviRksNV76EjSUfFMjdiFC5Qmwm2zKZMSjHeTK) - is that what you intended?

Reply:
Yes we were testing Pikespeak and they don't have as far i know testnet enviroment, I just deployed again potining to testnet. Thanks